src/infill/SubDivCube.cpp

```diff
@@ -226,8 +226,7 @@ int SubDivCube::distanceFromPointToMesh(SliceMeshStorage& mesh, int layer_nr, Po
         return 2;
         *distance2 = 0;
     }
-    Polygons collide;
-    mesh.layers[layer_nr].getInnermostWalls(collide, 2, mesh);
+    Polygons& collide = mesh.layers[layer_nr].getInnermostWalls(2, mesh);
```

Comment:
I know it was already there, but the "2" is a bit of a magic number. Could you add some documentation as to why the 2 is used?